### DISCUSSION TITLE
Registration not possible

### DISCUSSION THREAD

**Discussion by Lister0fSmeg** [Open-Shell/Open-Shell-Menu#1291] [General]
I have a question about Open Shell that I wanted to ask on the Classic Shell forum. I didn't think that it really belonged here TBH. I attempted to register for the forum but the Confirmation Code entry does not exist and therefore registration cannot be completed. Is that forum not used? There are a few posts including updates even though many are from last year. BTW I have tried three different browsers including Firefox Nightly and Opera which are very recent and contain no website blocking.

Have registrations been stopped on that forum?

**Comment by ge0rdi** [comment]
Original `Classic Shell` author wrote that he will keep the forum running for some time (but that was back in 2017).

http://www.classicshell.net/forum/viewtopic.php?f=4&t=8147
> The forum on http://www.classicshell.net/forum/ will stay open until the end of 2018, however I will not frequently participate in the discussions.

> Have registrations been stopped on that forum?

Most likely yes. Honestly, I'm surprised that the forum even works as the author needs to pay for storage/hosting all those years ...

**Reply by Lister0fSmeg** [reply]
Thanks for the reply and information.

Can a notice be put onto the forum, maybe at the top of the registration or somewhere where it can be clearly seen to inform everyone that the registrations are closed? I wasted a fair amount of time attempting to register. My main Sandboxed browser is fairly old and some sites don't work well with it and the ad blocking can also interfere with sites. Trying 2 other browsers also took time only to find the forum registration was not working. Simple notice which is easy to see would of avoided this.

**Reply by ge0rdi** [reply]
I'm not sure we (`Open-Shell` team) can help with it.
We have nothing to do with the original `www.classicshell.net` site, nor we are in touch with original `Classic Shell` author.

**Comment by Lister0fSmeg** [comment]
Oh IC.

I have checked and the last post was a Spam post only 2 days back which I thought might of been an update of the build seeing it was in the updates thread until I actually checked it. Previous to that a real post was made in November last year.

I have a question about using it because I am having problems with it. Where could I ask that? The damn explorer toolbar keeps going over to the left and cannot be moved. I did read an article about fixing it but it did not work for me. I have uninstalled and reinstalled it and rebooted. I have tried everything to shift the explorer toolbar buttons but I cannot get it to move. I prefer it on the right side.

**Comment by ge0rdi** [comment]
> Where could I ask that?

If you have questions then you can ask here in [Discussions](https://github.com/Open-Shell/Open-Shell-Menu/discussions/categories/q-a).
Ideally create one topic per one question.

If you think there is some problem with `Open-Shell` (that you can replicate, ideally), feel free to report an [issue](https://github.com/Open-Shell/Open-Shell-Menu/issues).
